azure-monitor-opentelemetry-distro/azure/monitor/opentelemetry/distro/constants.py

```diff
@@ -0,0 +1,70 @@
+from os import environ
+from pathlib import Path
+from sys import platform
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+_LOG_PATH_LINUX = "/var/log/applicationinsights"
+_LOG_PATH_WINDOWS = "\\LogFiles\\ApplicationInsights"
+# TODO: Enabled when duplciate logging issue is solved
+# __EXPORTER_DIAGNOSTICS_ENABLED_ENV_VAR = (
+#     "AZURE_MONITOR_OPENTELEMETRY_DISTRO_ENABLE_EXPORTER_DIAGNOSTICS"
+# )
+
+
+def _get_log_path(status_log_path=False):
+    if platform.startswith("linux"):
+        return _LOG_PATH_LINUX
+    elif platform.startswith("win32"):
+        log_path = str(Path.home()) + _LOG_PATH_WINDOWS
```

Comment:
Nit: Use `str.format()` instead of `+`

Reply:
Seems like using format() would be messier:
Would involve changing __LOG_PATH_WINDOWS to "{}\\LogFiles\\ApplicationInsights" and then saying __LOG_PATH_WINDOWS.format(str(Path.home())). I could also do "".join(str(Path.home()), __LOG_PATH_WINDOWS). But since this is simple concatenation, not string replacement, I think + makes the most sense.